Scripts/script-StixParser.yml

```diff
@@ -3,66 +3,323 @@ commonfields:
   version: -1
 name: StixParser
 script: |-
-  from stix.core import STIXPackage
   import json
+  import re
   import tempfile
-  with tempfile.NamedTemporaryFile() as temp:
-      temp.write(demisto.args()['iocXml'].encode('utf-8'))
-      temp.flush()
-      stix_package = STIXPackage.from_xml(temp.name)
-  data = []
-  i = 0
-  def create_new_ioc(data, i, time, pkg_id, ind_id):
+  from datetime import datetime
+
+  from stix.core import STIXPackage
+
+  """ HELPER FUNCTIONS"""
+
+
+  def convert_to_json(string):
+      """Will try to convert given string to json.
+
+      Args:
+          string: str of stix/json file. may be xml, then function will fail
+
+      Returns:
+          json object if succeed
+          False if failed
+      """
+      try:
+          js = json.loads(string)
+          return js
+      except ValueError:
+          return False
```

Comment:
Maybe return `None` instead?

Reply:
It'll result exactly the same behavior. but fine